tests/build.go

```diff
@@ -533,13 +534,19 @@ func testBuildRef(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err)
 
 	type mdT struct {
-		BuildRef string `json:"buildx.build.ref"`
+		BuildRef        string `json:"buildx.build.ref"`
+		BuildProvenance string `json:"buildx.build.provenance"`
 	}
 	var md mdT
 	err = json.Unmarshal(dt, &md)
 	require.NoError(t, err)
 
 	require.NotEmpty(t, md.BuildRef)
+	require.NotEmpty(t, md.BuildProvenance)
+
+	var prv slsa02.ProvenancePredicate
+	require.NoError(t, json.Unmarshal([]byte(md.BuildProvenance), &prv))
+	require.Equal(t, "https://mobyproject.org/buildkit@v1", prv.BuildType)
```

Comment:
`buildType` is hardcoded for testing atm as importing `github.com/moby/buildkit/solver/llbsolver/provenance` brings a lot of dependencies. I think we should move provenance types to a dedicated package on BuildKit.